orttraining/orttraining/test/training_ops/cuda/cross_entropy_test.cc

```diff
@@ -1073,5 +1069,25 @@ TEST(CrossEntropyTest, SoftmaxCrossEntropyLossInternalGrad_TinySizeTensorFloatIn
                                                             "none", 0, 5e-2, true);
 }
 
+#ifndef _WIN32
+// Disable the large size tests for Windows because it is too slow, running on Linux would be enough.
+TEST(CrossEntropyTest, SoftmaxCrossEntropyLossInternalGrad_LargeSizeTensorUInt64Index) {
+  // The element count is bigger than the upper limit of int32_t.
+  constexpr int64_t bsz = 419431;
+  constexpr int64_t vocab_size = 5120;
+  std::vector<int64_t> dY_dims{};
+  std::vector<int64_t> log_prob_dims{bsz, vocab_size};
```

Comment:
This allocates a 16 GB vector. Is that ok to run in the pipeline?

Reply:
Yes it can run in current pipeline. It takes around 3 minutes for the two tests - one for fw, one for bw. If it is not stable, we can consider to disable it. let me add a comment for it.